src/Modules/Render/ViewScene.cc

```diff
@@ -117,11 +117,22 @@ void ViewScene::setStateDefaults()
   state->setValue(Light2Inclination, 90);
   state->setValue(Light3Inclination, 90);
   state->setValue(ShowViewer, false);
+  state->setValue(CameraDistance, 3.0);
+  state->setValue(CameraLookAt, makeAnonymousVariableList(0.0, 0.0, 0.0));
+  state->setValue(CameraRotation, makeAnonymousVariableList(1.0, 0.0, 0.0, 0.0));
 
   get_state()->connectSpecificStateChanged(Parameters::GeometryFeedbackInfo, [this]() { processViewSceneObjectFeedback(); });
   get_state()->connectSpecificStateChanged(Parameters::MeshComponentSelection, [this]() { processMeshComponentSelection(); });
 }
 
+void ViewScene::fireTransientStateChangeSignalForGeomData()
+{
+  //this is gross but I dont see any other way to fire the signal associated with geom data
+  auto transient = get_state()->getTransientValue(Parameters::GeomData);
+  auto geoms = transient_value_cast<GeomListPtr>(transient);
+  get_state()->setTransientValue(Parameters::GeomData, geoms, true);
```

Comment:
Instead of this we can provide a function on the state object. I'll think about it for next pull request

Reply:
So should I just leave this?

Reply:
Okay sounds good